frontend/util/hooks/useLandlordSuggestions.ts

```diff
@@ -0,0 +1,57 @@
+import {useEffect, useState} from 'react'
+
+export const useLandlordSuggestions = (landlord: string) => {
+	const [landlordSuggestions, setLandlordSuggestions] = useState<string[]>([])
+	const [isSearching, setIsSearching] = useState(false)
+
+	useEffect(() => {
+		let timer: NodeJS.Timeout
+		if (landlord) {
+			setIsSearching(true)
+			timer = setTimeout(() => {
+				getLandlordSuggestions()
+			}, 500)
+		}
+
+		return () => {
+			clearTimeout(timer)
+		}
+	}, [landlord])
+
+	const getLandlordSuggestions = async () => {
```

Comment:
I'm thinking this should maybe be moved into Next API pages. This way it'll match the rest of our backend requests.